plugins/optimization-detective/storage/class-od-url-metrics-post-type.php

```diff
@@ -243,13 +243,8 @@ public static function store_url_metric( string $slug, OD_URL_Metric $new_url_me
 		}
 
 		$post_data['post_content'] = wp_json_encode(
-			array_map(
-				static function ( OD_URL_Metric $url_metric ): array {
-					return $url_metric->jsonSerialize();
-				},
-				$group_collection->get_flattened_url_metrics()
-			),
-			JSON_UNESCAPED_SLASHES // No need for escaped slashes since not printed to frontend.
+			$group_collection->get_flattened_url_metrics(),
+			JSON_UNESCAPED_SLASHES // No need for escaping slashes since this JSON is not embedded in HTML.
```

Comment:
This is a bonus change referenced in https://github.com/WordPress/performance/issues/1156#issuecomment-2628819409